src/lib/PhpCsFixer/Config.php

```diff
@@ -10,182 +10,17 @@
 
 use PhpCsFixer\Config as ConfigBase;
 
+/**
+ * @deprecated 1.3.0 The "Config" class has been deprecated, will be removed in 2.0. Use RuleSetInterface or InternalConfigFactory instead.
+ */
```

Comment:
Can we provide rector rule for these deprecation?

Reply:
> Can we provide rector rule for these deprecation?

:ok_hand:  Created IBX-8366.